cdap-tms/src/main/java/co/cask/cdap/messaging/store/AbstractMessageTable.java

```diff
@@ -117,13 +117,26 @@ public void delete(TopicId topicId, long startTimestamp, short startSequenceId,
     delete(startRow, Bytes.stopKeyForPrefix(stopRow));
   }
 
+  protected long decodeTx(byte[] txPtr) {
+    return Bytes.toLong(txPtr);
+  }
+
+  protected byte[] encodeTx(long txPtr) {
```

Comment:
After passing in txBytes, I removed those two methods. So we don't need to do this now.